crates/iota-sdk-types/src/hash.rs

```diff
@@ -435,7 +435,7 @@ impl crate::ObjectId {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(test, feature = "proptest"))]
```

Comment:
Not asking you to change but I guess I would have preferred putting them on individual tests, even if redundant.

Reply:
Why though if all the tests inside need it?